eng/common/scripts/check-spelling-in-changed-files.ps1

```diff
@@ -57,199 +57,14 @@ Param (
     [switch] $ExitWithError,
 
     [Parameter()]
-    [switch] $Test
+    [string]$SourceCommittish = "${env:SYSTEM_PULLREQUEST_SOURCECOMMITID}",
+
+    [Parameter()]
+    [string]$TargetCommittish = ("origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}" -replace "refs/heads/")
```

Comment:
These are needed to push values through to `Get-ChangedFiles` during testing. It also makes it possible to run the tool and tests locally again.

Reply:
Not needed for this PR work but we will eventually switch to using a pattern similar to https://github.com/Azure/azure-rest-api-specs/blob/main/eng/scripts/ChangedFiles-Functions.ps1#L1 for the Get-ChangedFiles.